spec/overview.html

```diff
@@ -32,7 +32,7 @@ <h1>API Overview</h1>
         Directly, by using a service constructor to construct an object, specifying a list of preferred languages and options to configure its behaviour. The object provides a main function (`compare`, `select`, `format`, etc.), which can be called repeatedly. It also provides a `resolvedOptions` function, which the application can use to find out the exact configuration of the object.
       </li>
       <li>
-        Indirectly, by using the functions of the ECMAScript 2022 Language Specification mentioned above. The collation and formatting functions are respecified in this specification to accept the same arguments as the Collator, NumberFormat, and DateTimeFormat constructors and produce the same results as their compare or format methods. The case conversion functions are respecified to accept a list of preferred languages.
+        Indirectly, by using the functions of the ECMAScript 2022 Language Specification mentioned above. The collation and formatting functions are respecified in this specification to accept the same arguments as the Collator, NumberFormat, RelativeTimeFormat and DateTimeFormat constructors and produce the same results as their compare or format methods. The case conversion functions are respecified to accept a list of preferred languages.
```

Comment:
Is `RelativeTimeFormat#format` called in any ecma262 functions? I couldn't recall any but just wanted to ask.

Reply:
Was added based on https://github.com/tc39/ecma402/issues/468#issue-644449434

Reply:
This section defines how applications indirectly use ECMA-402 through ECMA-262 overrides (described at https://tc39.es/ecma402/#locale-sensitive-functions , none of which reference part of RelativeTimeFormat).
```suggestion
        Indirectly, by using the functions of the ECMAScript 2022 Language Specification mentioned above. The collation and formatting functions are respecified in this specification to accept the same arguments as the Collator, NumberFormat, and DateTimeFormat constructors and produce the same results as their compare or format methods. The case conversion functions are respecified to accept a list of preferred languages.
```